src/main/resources/db/migration/V202409081445__create_owner_table.sql

```diff
@@ -0,0 +1,20 @@
+create table owner
+(
+    id serial primary key,
+    name varchar(100) not null,
+    company_id int,
+    employee_id varchar(100) not null
+);
+CREATE INDEX idx_name_owner_employee ON owner(owner_id, employee_id);
```

Comment:
dont forget spaces
and the same comment as before
the index name should be indicative, for example - idx_pets_company_id